docs/articles/testkube-dashboard-workflow-editor.md

```diff
@@ -1,81 +1,87 @@
+![Save & Run](../articles/images/workflow-editor-save-run.png)
 
-![Editor Popup Menu](../img/editor-popup-menu.png)
+When you select **Save & Run**, you can view the [Execution Details](../articles/testkube-dashboard-execution-details.md) in the modal that opens. From this modal, you may also **Run again**.
 
-Once you're happy with your changes, use the Save & Run option to the top right to immediately trigger an execution
-after saving, the corresponding Execution Details will be opened automatically.
+![Save & Run Results](../articles/images/workflow-editor-save-run-results.png)
 
 ## Template Inlining
 
-The below screenshot shows the Inline-Templates view in Full-screen mode:
-- the left panel shows the actual workflow with the `use` clause
-- the right part shows the steps inlined by the `official--artillery--v1` template
+This tool allows you to preview two versions of the same YAML file: one (left) is abbreviated; the other (right) displays a more complete and extended YAML version. Re-select 
+**Template Inlining** to remove these previews.
 
-![Test Workflow editor in fullscreen with inline diff](images/workflows-fullscreen-inline-diff.png)
+![Inline Template](../articles/images/workflow-editor-inline-template.png)
 
-## Workflow Specific Editor Intelligence
+## Workflow-Specific Editor Intelligence
 
-The Workflow YAML editor in the Definition panel described above the has a number of 
-features that help you work specifically with Workflow YAML:
+The **Workflow Editor** includes several features.
 
-### Documentation Hovers 
+### Hover Documentation 
 
-Helps you understand the purpose/usage of the corresponding Workflow property.
+This feature helps you understand the purpose and usage of the corresponding Workflow property.
 
 ![Editor Documentation Hover](../img/editor-documentation-hover.png)
 
 ### Autocomplete for Applicable Properties
 
-Helps you craft valid Workflow YAMLs
+This feature predicts or suggests YAML keywords or values as you type, allowing you to select the suggestion(s) as appropriate.
 
-![Workflow Properties Autocomplete](../img/workflow-properties-autocomplete.png)
+![Workflow Properties Autocomplete](../articles/images/workflow-editor-auto-complete.png)
 
 ### Error Indicators for Invalid Properties
 
-Helps you find and correct Workflow YAML errors, in the below screenshot the `service` property is marked
-(it should be `services`)
+The feature adds an underline (sometimes called an *error squiggle*) below a keyword or **Property** which indicates, for example, a misspelling. This feature allows you to readily identify and correct this and similar errors.
 
 ![Workflow Editor Error Marker](images/workflow-editor-error-marker.png)
 
 ### Autocomplete for Template Names
 
-Helps you select valid template(s) used in your Workflows wherever a template name is expected:
+This feature helps you select valid template(s) used in your Workflows, particularly when you are expected to enter a Workflow template name.
```

Comment:
I don't think "particularly when you are expected to enter a Workflow template name." is needed - there is no other use-case for this feature so just "This feature helps you select valid template(s) using in your Workflows." is enough!?